wallet/bench/config/Endpoints.csv

```diff
@@ -0,0 +1,4 @@
+BenchName,BenchDuration,MinDelayBetweenCalls,MaxDelayBetweenCalls,PathToReportFile
+GetHistoryBench,5.0,0.5,2.5,/tmp/GetHistoryBenchReport.csv
```

Comment:
This assumes the target environment has a $TMPDIR, but what about Windows? 😉  My recommendation is to create a local directory (which we `gitignore`) that stores these results.

Reply:
No, please don't look at the content of this file. :-) These lines were added just to keep this file nonempty. Of course these values will change.